src/components/structures/LoggedInView.js

```diff
@@ -434,17 +434,21 @@ const LoggedInView = React.createClass({
         }
 
         const mauLimitEvent = this.state.serverNoticeEvents.find((e) => {
```

Comment:
As MAU is now just one case of resource limit exceeded, how about renaming the variables that are not specific to MAU?